formats/policy/frontend/schema.json

```diff
@@ -99,6 +99,9 @@
                     }
                   }
                 }
+              },
+              "result_metadata_name_key": {
```

Comment:
This is probably better off grouped with the `display_as_result_metadata` key a few properties up.